packages/vitest/src/runtime/child.ts

```diff
@@ -58,10 +59,12 @@ function unwrapConfig(config: ResolvedConfig) {
 }
 
 export async function run(ctx: ChildContext) {
+  const inspectorCleanup = setupInspect(ctx.config)
   init(ctx)
   const { run, executor } = await startViteNode(ctx)
   await run(ctx.files, ctx.config, ctx.environment, executor)
   await rpcDone()
+  inspectorCleanup()
```

Comment:
Do we need to clean it up, if run fails? `try/finally`

Reply:
Good point. An open inspector can also prevent the worker thread from terminating.

Just to be sure that inspector does not remain open, I'll wrap all these functions inside the `try`.